packages/@n8n/nodes-langchain/nodes/llms/LmChatGoogleVertex/LmChatGoogleVertex.node.ts

```diff
@@ -97,11 +97,13 @@ export class LmChatGoogleVertex implements INodeType {
 				const results: Array<{ name: string; value: string }> = [];
 
 				const credentials = await this.getCredentials('googleApi');
+				const privateKey = (credentials.privateKey as string).replace(/\\n/g, '\n').trim();
```

Comment:
Shouldn't this use `formatPrivateKey` from `@utils/utilities`, as outlined in the ticket?